.travis.yml

```diff
@@ -0,0 +1,35 @@
+---
+sudo: required
+dist: trusty
+language: node_js
+
+node_js:
+  - 9
+python:
+  - 3.6.x
+
+addons:
+  apt:
+    packages:
+      - imagemagick
+      - graphicsmagick
+
+install:
+  - wget https://github.com/gohugoio/hugo/releases/download/v0.31.1/hugo_0.31.1_Linux-64bit.deb
```

Comment:
Is there still an issue with the current release of hugo and nats-site?

Reply:
yes there is.